src/test/java/com/wikia/webdriver/testcases/mobilewikitests/TrackingOptInModalTests.java

```diff
@@ -29,11 +29,11 @@
       dataProvider = "GDPRCountriesForTest"
   )
   public void testModalVisibilityForAnon(String continent, String country, boolean shouldGetModal) {
-    TrackingOptInModal.setGeoCookie(driver, continent, country);
+    TrackingOptInPage.setGeoCookie(driver, continent, country);
     new ArticlePage().open();
 
     PageObjectLogging.logInfo("Geo cookie: ", driver.manage().getCookieNamed("Geo").getValue());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "Geo cookie: " 12 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)